syntheseus/reaction_prediction/environments/setup_shared.sh

```diff
@@ -5,7 +5,7 @@ mkdir -p external
 
 # Add the `external/` directory to `PYTHONPATH` when the environment is activated.
 mkdir -p $CONDA_PREFIX/etc/conda/activate.d
-echo "export PYTHONPATH=$PYTHONPATH:$PWD/external" >> $CONDA_PREFIX/etc/conda/activate.d/env_vars.sh
+echo "export PYTHONPATH=.:$PWD/external" >> $CONDA_PREFIX/etc/conda/activate.d/env_vars.sh
```

Comment:
Doesn't this assume the user's `PYTHONPATH` is empty?

Reply:
It ignores the pre-existing contents of `PYTHONPATH`. I thought that would make the setup more predictable, but maybe we shouldn't be so aggressive 😄 Although I guess we could prepend our `$PWD/external` (instead of appending) just in case the pre-existing `PYTHONPATH` contains directories named in the same way as one of our single-step models...

Reply:
I think any of these options are fine (including overwriting the PYTHONPATH), just wanted to ensure it wasn't an oversight.

Reply:
I've now switched to prepending but keeping the current contents of `PYTHONPATH`.